core/controllers/admin_test.py

```diff
@@ -657,49 +658,52 @@ def test_regenerate_topic_related_opportunities_action(self) -> None:
         story_services.save_new_story(owner_id, story)
         topic_services.add_canonical_story(
             owner_id, topic_id, story_id)
-        topic_services.publish_story(topic_id, story_id, self.admin_id)
-        story_services.update_story(
-            owner_id, story_id, [story_domain.StoryChange({
-                'cmd': 'add_story_node',
-                'node_id': 'node_1',
-                'title': 'Node1',
-            }), story_domain.StoryChange({
-                'cmd': 'update_story_node_property',
-                'property_name': 'exploration_id',
-                'node_id': 'node_1',
-                'old_value': None,
-                'new_value': '0'
-            })], 'Changes.')
 
-        all_opportunity_models = list(
-            opportunity_models.ExplorationOpportunitySummaryModel.get_all())
-
-        self.assertEqual(len(all_opportunity_models), 1)
-
-        old_creation_time = all_opportunity_models[0].created_on
-
-        self.login(self.CURRICULUM_ADMIN_EMAIL, is_super_admin=True)
-        csrf_token = self.get_new_csrf_token()
-
-        result = self.post_json(
-            '/adminhandler', {
-                'action': 'regenerate_topic_related_opportunities',
-                'topic_id': 'topic'
-            }, csrf_token=csrf_token)
+        with self.swap_to_always_return(
```

Comment:
There's a new (and much simpler) way to handle feature flag testing, please see https://github.com/oppia/oppia-web-developer-docs/pull/262 and here: https://github.com/oppia/oppia/pull/19756. Could you please update this and other usages accordingly?

Thanks!

Reply:
Done.